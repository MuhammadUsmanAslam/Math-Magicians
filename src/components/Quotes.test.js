import renderer from "react-test-renderer";
import Quotes from "./Quotes";

test("render quote component", () => {
	const tree = renderer.create(<Quotes></Quotes>).toJSON();

	expect(tree).toMatchSnapshot();
});
