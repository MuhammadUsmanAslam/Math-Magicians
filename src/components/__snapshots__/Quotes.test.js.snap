// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render quote component 1`] = `
<div
  className="quotes"
>
  <div
    className="spinner-box"
  >
    <div
      className="configure-border-1"
    >
      <div
        className="configure-core"
      />
    </div>
    <div
      className="configure-border-2"
    >
      <div
        className="configure-core"
      />
    </div>
  </div>
</div>
`;
